spyder/api/plugins.py

```diff
@@ -282,6 +284,18 @@ def get_plugin_icon(self):
         """
         return ima.icon('outline_explorer')
 
+    def set_plugin_icon_size(self, iconsize):
+        """
+        Set the icon size of plugin.
+
+        This must be reimplemented by plugins that need to adjust the icon
+        size of their toolbar.
+        """
+        try:
+            self.options_button.setIconSize(QSize(iconsize, iconsize))
+        except AttributeError:
```

Comment:
Why do we need the try/except? An example in a comment could add clarity to this section of code

Reply:
That was me being lazy.

I did it to avoid refactoring the Project and Breakpoints plugin layout, since their toolbar only add the options button. Anyway, since we are making things uniform, I did it in the end and so I removed this confusing implementation from the API.